### PR TITLE
[API] Add Product Image type to responses

### PR DESCRIPTION
**Pull request by arti0090**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

**Review by AdamKasp — APPROVED** [review]

File: src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ProductImage.xml [hunk 1]
```diff
@@ -6,10 +6,20 @@
...
+            <group>admin:product_image:read</group>
+            <group>shop:product_image:read</group>
             <group>admin:product:read</group>
             <group>shop:product:read</group>
         </attribute>
         <attribute name="path">
+            <group>admin:product_image:read</group>
+            <group>shop:product_image:read</group>
+            <group>admin:product:read</group>
+            <group>shop:product:read</group>
```
Comment (AdamKasp) [inline src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ProductImage.xml:18]: why do you add this serialization also on product response?

File: tests/Api/Admin/ProductImageTest.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\Api\Admin;
+
+use Sylius\Component\Core\Model\ProductImageInterface;
+use Sylius\Tests\Api\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+final class ProductImageTest extends JsonApiTestCase
```
Comment (GSadee) [inline tests/Api/Admin/ProductImageTest.php:20]:
```suggestion
final class ProductImagesTest extends JsonApiTestCase
```

File: tests/Api/Admin/ProductImageTest.php [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+        $this->assertResponse($response, 'admin/get_product_images_response', Response::HTTP_OK);
+    }
+
+    /** @test */
+    public function it_gets_one_product_image(): void
+    {
+        $fixtures = $this->loadFixturesFromFiles(['product_image.yaml', 'authentication/api_administrator.yaml']);
+        /** @var ProductImageInterface $productImage */
+        $productImage = $fixtures["product_thumbnail"];
```
Comment (GSadee) [inline tests/Api/Admin/ProductImageTest.php:69]:
```suggestion
        $productImage = $fixtures['product_thumbnail'];
```

File: tests/Api/Shop/ProductImageTest.php [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\Api\Shop;
+
+use Sylius\Component\Core\Model\ProductImageInterface;
+use Sylius\Tests\Api\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+final class ProductImageTest extends JsonApiTestCase
```
Comment (GSadee) [inline tests/Api/Shop/ProductImageTest.php:20]:
```suggestion
final class ProductImagesTest extends JsonApiTestCase
```

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, @arti0090! :tada:

**Pull request merged**